recipes/django/meta.yaml

```diff
@@ -0,0 +1,233 @@
+{% set version = "1.8.11" %}
+
+package:
+    name: django
+    version: {{ version }}
+
+source:
+    fn: Django-{{ version }}.tar.gz
+    url: https://pypi.python.org/packages/source/D/Django/Django-{{ version }}.tar.gz
+    md5: 1c85d0d582dc211adc6851dd2dc86228
+    patches:
+        - libgdal.patch
```

Comment:
Why can't the code find it in conda? Is there something conda could do so that these packages do find things appropriately?

I don't like patching so many things for the same reason. If conda can be fixed, that seems better to me.